### PR TITLE
485/handle missing binaries

### LINKED ISSUE #485: If Monkey agent binaries are missing, Monkey Island shows no indication
**Describe the bug**
When the Monkey agent binary is missing, the Island shows no clear indication as to WHY it is failing to run, instead only producing this log: 
```
2019-11-10 16:12:16,361 - local_run.py:41 - run_local_monkey() - ERROR - Copy file failed
Traceback (most recent call last):
  File "/var/monkey_island/monkey_island/cc/resources/local_run.py", line 38, in run_local_monkey
    copyfile(monkey_path, target_path)
  File "/home/user/projects/monkey/lib/python3.6/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/var/monkey_island/monkey_island/cc/binaries/monkey-linux-64'
2019-11-10 16:12:16,361 - wsgi.py:196 -       _log() - INFO - 200 POST /api/local-monkey (::1) 1.63ms
```

**To Reproduce**
1. Set up Monkey Island without the Monkey Binaries
2. Press "Run on Monkey Island"
3. Nothing happens
4. See error in logs

**Expected behavior**
The Island should pop up an error message to the user explaining what happened and how to fix it/contact us.

**Screenshots/logs**
```
2019-11-10 16:12:16,361 - local_run.py:41 - run_local_monkey() - ERROR - Copy file failed
Traceback (most recent call last):
  File "/var/monkey_island/monkey_island/cc/resources/local_run.py", line 38, in run_local_monkey
    copyfile(monkey_path, target_path)
  File "/home/user/projects/monkey/lib/python3.6/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/var/monkey_island/monkey_island/cc/binaries/monkey-linux-64'
2019-11-10 16:12:16,361 - wsgi.py:196 -       _log() - INFO - 200 POST /api/local-monkey (::1) 1.63ms
```

Thanks @liad-guardicore for letting us know

### PR DESCRIPTION
**Pull request by ophirharpazg**
# What is this? 
Fixes #485 . 

Add any further explanations here. 

## Checklist
* [X] Have you added an explanation of what your changes do and why you'd like to include them?
* [X] Have you successfully tested your changes locally?
* [x] Is the TravisCI build passing? 

## Proof that it works
If applicable, add screenshots or log transcripts of the feature working
image

## Changes
Are the commit messages enough? If not, elaborate.

### CODE REVIEW

File: monkey/monkey_island/cc/ui/src/components/ui-components/MissingBinariesModal.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <Modal.Body>
+          <h3>
+            <div className='text-center'>Uh oh...</div>
+          </h3>
+          <div style={{'marginTop': '1em', 'marginBottom': '1em'}}>
+            <p className="alert alert-warning">
+              <i className="glyphicon glyphicon-info-sign" style={{'marginRight': '5px'}}/>
+              Some Monkey binaries are not found where they should be...<br/>
+              Try downloading them from <a href="https://github.com/guardicore/monkey/releases/latest" target="blank">here</a>,
+              at the bottommost section titled "Assets".
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/src/components/ui-components/MissingBinariesModal.js:38]: Ideally we should add "... and place them in `monkey/monkey_island/cc/binaries` directory."

Comment (ophirharpazg) [reply monkey/monkey_island/cc/ui/src/components/ui-components/MissingBinariesModal.js]: I wish I could make the directory path look like `that`, I used italics instead 🙄

Comment (ShayNehmad) [reply monkey/monkey_island/cc/ui/src/components/ui-components/MissingBinariesModal.js]: You can do `this` using `import marked from 'marked';` and then styling your text using  Markdown. You can see an example of this in `monkey_island/cc/ui/src/components/attack/techniques/MitigationsComponent.js`.

https://github.com/markedjs/marked

Comment (ShayNehmad) [reply monkey/monkey_island/cc/ui/src/components/ui-components/MissingBinariesModal.js]: Alternatively, use a `div` with the `className="label"`

Comment (ophirharpazg) [reply monkey/monkey_island/cc/ui/src/components/ui-components/MissingBinariesModal.js]: I simply used `<code>` :)
image

**Pull request merged**